src/lib/learn-client/api/progress/formatting.ts

```diff
@@ -38,3 +38,41 @@ export function progressStatusToPercent(
 		return TutorialProgressPercent.Zero
 	}
 }
+
+/**
+ * Map TutorialProgressStatus to text for display use.
+ */
+const progressLabels: Record<TutorialProgressStatus, string> = {
+	visited: 'Visited',
+	in_progress: 'In progress',
+	complete: 'Complete',
+}
+
+/**
+ * Given a TutorialProgressStatus,
+ * Returns text representing the status suitable for display.
+ */
+export function progressStatusToLabel(
+	progressState: TutorialProgressStatus
+): string {
+	return progressLabels[progressState]
+}
+
+/**
+ * Map TutorialProgressStatus to text for screen reader use.
+ */
+const progressAriaLabels: Record<TutorialProgressStatus, string> = {
+	visited: 'Tutorial is visited.',
+	in_progress: 'Tutorial is in progress.',
+	complete: 'Tutorial is complete.',
+}
+
+/**
+ * Given a TutorialProgressStatus,
+ * Returns text representing the status suitable for screen reader use.
+ */
+export function progressStatusToAriaLabel(
```

Comment:
I needed utilities to get from a progress status to separate labels, one suitable for display and one suitable for screen readers. Felt natural to include them here in `formatting`.